ui/lib/pki/addon/routes/certificates.js

```diff
@@ -0,0 +1,3 @@
+import Route from '@ember/routing/route';
+
+export default class CertificatesRoute extends Route {}
```

Comment:
Just a nit on class naming here. In the case that an error is thrown it might be nice to see a more specific name in the stack trace like `PkiCertificatesRoute`.